msg/OpenDroneIdSystem.msg

```diff
@@ -0,0 +1,15 @@
+uint64 timestamp
+uint8 target_system
+uint8 target_component
+uint8[20] id_or_mac
+uint8 operator_location_type
+uint8 classification_type
+uint32 operator_latitude
```

Comment:
The mavlink definition has this as an int32_t. Makes sense as LAT can be neg. in southern hemisphere

Reply:
Fixed